pom.xml

```diff
@@ -1,102 +1,128 @@
-
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.8.0-02</version>
+    <version>3.19.0-SNAPSHOT</version>
   </parent>
-  <groupId>org.sonatype.nexus.plugins</groupId>
-  <artifactId>nexus-repository-p2</artifactId>
-  <name>${project.groupId}:${project.artifactId}</name>
-  <version>0.0.2</version>
-  <inceptionYear>2017</inceptionYear>
-  <packaging>bundle</packaging>
 
-  <dependencies>
-
-    <dependency>
-      <groupId>org.sonatype.nexus</groupId>
-      <artifactId>nexus-plugin-api</artifactId>
-      <scope>provided</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.sonatype.nexus</groupId>
-      <artifactId>nexus-repository</artifactId>
-      <scope>provided</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-compress</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>org.sonatype.goodies</groupId>
-      <artifactId>goodies-testsupport</artifactId>
-      <scope>test</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.sonatype.nexus</groupId>
-      <artifactId>nexus-rapture</artifactId>
-      <scope>provided</scope>
-    </dependency>
+  <artifactId>p2-parent</artifactId>
+  <version>0.0.2</version>
+  <packaging>pom</packaging>
+  <name>${project.groupId}:${project.artifactId}-parent</name>
 
-    <dependency>
-      <groupId>org.xmlunit</groupId>
-      <artifactId>xmlunit-core</artifactId>
-      <scope>test</scope>
-    </dependency>
+  <inceptionYear>2017</inceptionYear>
 
-    <dependency>
-      <groupId>org.tukaani</groupId>
-      <artifactId>xz</artifactId>
-    </dependency>
-  </dependencies>
+  <modules>
+    <module>nexus-repository-p2</module>
+    <module>nexus-repository-p2-it</module>
+  </modules>
+
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+  </distributionManagement>
+
+  <properties>
+    <nxrm-version>3.19.0-SNAPSHOT</nxrm-version>
+  </properties>
+
+  <repositories>
+    <!-- ensure we can find the parent pom when starting from an empty local .m2 repository -->
+    <repository>
+      <id>rso-snapshots</id>
+      <url>https://repository.sonatype.org/content/repositories/snapshots</url>
+    </repository>
+  </repositories>
 
   <build>
     <plugins>
+      <!--
```

Comment:
is this comment is necessary? we should have in README prerequisites information about java. 
not sure it is good to keep such information on both places.

Reply:
@DarthHater @bhamail